### ISSUE TITLE
Convert to a CLI/npm module

### ISSUE THREAD

**Issue by tkadlec** [abdulsuhail/wpt-crawler#1]
This is already super helpful, but I think it opens up more possibilities and probably better overall ergonomics if this was available to run like this (for example):

`index.js -level 3 -url_limit 10 -key #### -whiteListUrls whiteListUrls.txt`

A few of those could take either config files or just params (like, whiteListUrls could potentially be just a list of urls passed directly, and I could see a config param to pass in a config file versus using `level`, `url_limit`, etc.

The end goal though is that when this is set, we could wrap this up as an npm module so people could eventually have something like:

`npx wpt-crawler -level 3 .....`

Which would make it easier to drop into other workflows. 

What do you think, @abdulsuhail?

**Comment by abdulsuhail** [comment]
yeah, totally makes sense. This would make lives much easier, will work on it.

**Comment by Siddhantshukla814** [comment]
The Issue has already been resolved and can be closed
